tests/test_fields.py

```diff
@@ -451,3 +451,15 @@ async def name(self):  # noqa: D102
 
     page = Page()
     assert await page.name == "namex"
+
+
+def test_field_mixin() -> None:
+    class Mixin:
+        @field
+        def a(self):
+            return "a"
+
+    class A(ItemPage, Mixin):
+        pass
```

Comment:
Could you please change the test to check that A class can define fields as well? Probably it also makes sense to check that A can inherit from 2 base classes which define fields.